### ISSUE TITLE
Allow tilde expansion for fields in .linesrc

### ISSUE THREAD

**Issue by memmerto** [lifelines/lifelines#394]
Tilde expansion in fields in .linesrc (and probably other places) does not work.
This is unexpected and confusing and should be fixed.
(See LINES-L on 2020-04-05 for an example.)

**Comment by tfmorris** [comment]
I suspect "unexpected" is a generational thing. Traditionally, tilde expansion is done by the shell on Unix systems and basic C programs don't understand the concept.

As an aside, some (all?) implementations of libc's `wordexp` actually invoke `/bin/sh` in a subprocess to get this expansion done, so not super fast.

**Comment by memmerto** [comment]
@tfmorris I agree.  I suspect I was feeling excessively lazy when opening this issue and probably responded incorrectly on the list.

I would pivot this request to "warn when tildes or variables are used in .linesrc".  This should resolve the issue of users using `~/some/dir` or `$HOME/some/dir` in their config files.
